src/isomorphic/ReactDebugTool.js

```diff
@@ -58,17 +58,38 @@ var ReactDebugTool = {
   onSetState() {
     emitEvent('onSetState');
   },
-  onMountRootComponent(internalInstance) {
-    emitEvent('onMountRootComponent', internalInstance);
+  onSetIsTopLevelWrapper(debugID, isTopLevelWrapper) {
+    emitEvent('onSetIsTopLevelWrapper', debugID, isTopLevelWrapper);
   },
-  onMountComponent(internalInstance) {
-    emitEvent('onMountComponent', internalInstance);
+  onSetIsComposite(debugID, isComposite) {
+    emitEvent('onSetIsComposite', debugID, isComposite);
   },
-  onUpdateComponent(internalInstance) {
-    emitEvent('onUpdateComponent', internalInstance);
+  onSetDisplayName(debugID, displayName) {
+    emitEvent('onSetDisplayName', debugID, displayName);
   },
-  onUnmountComponent(internalInstance) {
-    emitEvent('onUnmountComponent', internalInstance);
+  onSetChildren(debugID, childDebugIDs) {
```

Comment:
We shouldn't this now but as an FYI, I been thinking that the notion of a "child" component isn't limited to children. It's not in React components so why should it be in native.

AirBnB wanted to do this `<input placeholder={<CustomComponent />} />` for example which would have one of the attributes expand into a series of "children".

Similar things can happen with layout I think. Anyway, let's fix that later when we know when it happens.